### PR TITLE
Add clarification note and contents detail

### LINKED ISSUE #2: What is this repo for?
Is it

1. the specific deployment files for beta.mybinder.org?
2. example/reference/documentation on which people should base their own deployments?
3. something else?

It's not clear from the readme. I had assumed 1., which would suggest that it shouldn't be part of people putting together their own deployments. But @choldgraf mentioned this repo when going through a binder deployment, so now I'm not sure.

cc @willingc

### PR DESCRIPTION
**Pull request by willingc**
- Add note directing users to the more general BinderHub repo/docs
- Add some description for files (@yuvipanda, @choldgraf I'm sure that I've described some of these incorrectly so please clarify as needed. thx.

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,33 @@
...
-This repo contains config files for different steps in a deployment workflow:
+This repo contains config files used in the deployment workflow
+for `beta.mybinder.org`.
+
+### `binder` directory
 
 - `common.yaml` configuration settings that are used in all deployment files
-- `local.yaml` (TODO)
-- `staging.yaml` config used for staging a BinderHub deployment for testing
+- `staging.yaml` config used for staging the `beta.mybinder.org` deployment
```
Comment (yuvipanda) [inline README.md]: It's the config for the 'staging' binder service, which currently lives at https://binder.binder-staging.omgwtf.in/ (and should move to staging.mybinder.org at some point)

File: README.md [hunk 2]
```diff
@@ -2,15 +2,33 @@
...
+### `support` directory
+
+- `Chart.yaml` identifies name, API version, and version of the chart
+- `requirements.yaml` identifies third party dependencies, such as
+  prometheus, grafana, and others, and their specific versions
+- `values.yaml` config values for third party tools
+
+### repo root directory
+
+- `common.yaml` (TODO: How does this differ from the common.yaml in the binder directory)
```
Comment (yuvipanda) [inline README.md]: This file should just be removed.

File: README.md [hunk 3]
```diff
@@ -2,15 +2,33 @@
...
-- `staging.yaml` config used for staging a BinderHub deployment for testing
+- `staging.yaml` config used for staging the `beta.mybinder.org` deployment
+  for testing
+- `beta.yaml` config used for the beta deployment
 - `prod.yaml` config used in a production deployment
 
-[binderhub-deploy]: https://github.com/jupyterhub/binderhub-deploy
-[BinderHub]: https://github.com/jupyterhub/binderhub
-[JupyterHub]: https://github.com/jupyterhub/jupyterhub
+### `support` directory
```
Comment (yuvipanda) [inline README.md]: This contains a set of 3rd party charts we use for supporting our own code on the binder deployment. The charts we use and their versions are specified in requirements.yaml, and the configuration of those charts is in values.yaml.

**Comment by yuvipanda** [comment]
I left some comments but otherwise looks good, @willingc! Tyvm!

File: README.md (hunk 3 above)
Comment (willingc) [reply README.md]: Added this content.

**Comment by yuvipanda** [comment]
tyvm @willingc!

**Pull request merged**